Lecture2/Task3/TaskThreeMain.java

```diff
@@ -0,0 +1,62 @@
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class TaskThreeMain {
+	final static String FIRST_LETTER = "M";
+
+	public static void main(String[] args) {
+		List<Student> students = new ArrayList<>();
+
+		students.add(new Student(1, 60, "Rudolf", "Schenker"));
+		students.add(new Student(2, 75, "Klaus", "Meine"));
+		students.add(new Student(3, 55, "Matthias", "Jabs"));
+		students.add(new Student(4, 58, "Paweł", "Mąciwoda"));
+		students.add(new Student(5, 64, "Mikkey", "Dee"));
+
+		System.out.println("List sorted by Age:");
+		students = getSortByAge(students);
+        	students.forEach(System.out::println);
+
+		System.out.println(String.format("First letter of the name is %s:", FIRST_LETTER));
+		List<Student> studentsM = getStudentsM(students, FIRST_LETTER);
+		studentsM.forEach(System.out::println);
+
+		System.out.println(String.format("Average age of students is:\n%s",
+				getAverageAge(students)));
+
+		System.out.println("Names from the Map:");
+		Map<Integer, Student> studentsMap;
+        	getStudentsListToMap(students);
```

Comment:
you ignore getStudentsListToMap result and print initial list of students

Reply:
fixed